modules/videoio/src/cap_librealsense.cpp

```diff
@@ -10,20 +10,19 @@
 namespace cv
 {
 
-VideoCapture_LibRealsense::VideoCapture_LibRealsense(int index)
+VideoCapture_LibRealsense::VideoCapture_LibRealsense(int) : mAlign(RS2_STREAM_COLOR)
 {
     try
     {
-        mDev = mContext.get_device(index);
-        // Configure all streams to run at VGA resolution at 60 frames per second
-        mDev->enable_stream(rs::stream::depth, 640, 480, rs::format::z16, 60);
-        mDev->enable_stream(rs::stream::color, 640, 480, rs::format::bgr8, 60);
-        mDev->enable_stream(rs::stream::infrared, 640, 480, rs::format::y8, 60);
-        mDev->start();
+        rs2::config config;
+        // Configure all streams to run at VGA resolution at default fps
+        config.enable_stream(RS2_STREAM_DEPTH, 640, 480, RS2_FORMAT_Z16);
+        config.enable_stream(RS2_STREAM_COLOR, 640, 480, RS2_FORMAT_BGR8);
+        config.enable_stream(RS2_STREAM_INFRARED, 640, 480, RS2_FORMAT_Y8);
+        mPipe.start();
     }
-    catch (rs::error&)
+    catch (rs2::error&)
```

Comment:
Could you please "catch" by "const reference"? (here and below)

Link: http://www.riptutorial.com/cplusplus/example/9212/best-practice--throw-by-value--catch-by-const-reference